src/test/java/core/storage/StorageImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.storage;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class StorageImplTest {
+
+    private Storage storage;
+
+    @Before
+    public void setUp() {
+        storage = new StorageImpl();
+        storage.add("apple", 100);
+        storage.add("banana", 50);
+    }
+
+    @Test
+    public void storageImpl_getAllData_Ok() {
+        Map<String, Integer> expected = new HashMap<>();
+        expected.put("apple", 100);
+        expected.put("banana", 50);
+        Map<String, Integer> actual = storage.getAllData();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void storageImpl_addNewFruitAndCheck_Ok() {
+        Map<String, Integer> allDataBeforeAdd = storage.getAllData();
+        allDataBeforeAdd.put("coconut", 10);
+        storage.add("coconut", 10);
+        Map<String, Integer> allDataAfterAdd = storage.getAllData();
+        assertEquals(allDataBeforeAdd, allDataAfterAdd);
```

Comment:
better create excepted value yourself, don't use `storage.getAllData()` for expected